util/kustomize/testdata/kustomization_yaml/statefulset.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
```

Comment:
why was this file added?

Reply:
Sigh. Bad split. It's from the other side.

Reply:
Specifically, that commit was on both sides of this work...